_includes/head.html

```diff
@@ -27,4 +27,5 @@
   {% endif %}
   <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl | replace: '//', '/' }}">
   <link href="/atom.xml" type="application/atom+xml" rel="alternate" title="ATOM Feed" />
+  {% seo %}
```

Comment:
With this, you're set up and can start decorating your posts with the attributes. Check https://github.com/jekyll/jekyll-seo-tag#usage

I suggest to add the desired info to `_config.yml`, if it doesn't exist already.

I suggest to do that in this PR.